docs/compiler.md

```diff
@@ -153,10 +170,10 @@ events:
         type: ByteString
 ```
 
-It's passed to the `deploy` command via `-c` option:
+Then the manifest can be passed to the `deploy` command via `-m` option:
 
 ```
-$ ./bin/neo-go contract deploy -i contract.nef -c contract.yml -r http://localhost:20331 -w wallet.json -g 0.001
+$ ./bin/neo-go contract deploy -i contract.nef -m contract.manifest.json -r http://localhost:20331 -w wallet.json
```

Comment:
We should make file names unified with the previous command (either both `contract*` or `mycontract*`).

Reply:
Not sure it's really important, but I've changed everything to `contract*`.